resources/eventing/values.yaml

```diff
@@ -8,7 +8,7 @@ global:
       version: PR-11920
     eventing_controller:
       name: eventing-controller
-      version: PR-12553
+      version: PR-12596
```

Comment:
as this image was not created in the referenced image please use either the PR-Number of the other PR (the one that creates the new image) or if that one is already merged directly refer to the final commit sha of the merging commit of the pr that created the image

Reply:
done